### ISSUE TITLE
expand function and row height

### ISSUE THREAD

**Issue by sailfish009** [boussadjra/vueye-table#18]
[initial sate]
![image](https://user-images.githubusercontent.com/17582766/105801910-8a70ee00-5fdd-11eb-8e3f-7fc0b9933873.png)

[only one row in datatable and expanded it]
![image](https://user-images.githubusercontent.com/17582766/105801936-96f54680-5fdd-11eb-807f-ae4d3c20637c.png)

[row height is too high, when collapsed it]
![image](https://user-images.githubusercontent.com/17582766/105802037-c60bb800-5fdd-11eb-849c-c0e5ba390e09.png)

**Comment by boussadjra** [comment]
could you build a sandbox that reproduces the issue?

**Comment by sailfish009** [comment]
@boussadjra yes, here is sandbox: https://codesandbox.io/s/vueye-table-forked-hf96q

note that even if rows length is not 1 (employeetest.json), there is still height bugs exist.

for example, try to expand and close '>'  in page '2'

```
//import employees from "./assets/employees.json";    // rows length  > 1
import employees from "./assets/test.json";                  // rows length == 1
```

**Comment by boussadjra** [comment]
@sailfish009 thanks for raising this issue I'll work to debug it and fix it

**Comment by boussadjra** [comment]
@sailfish009 I can't see that issue, which browser are you using?

**Comment by sailfish009** [comment]
@boussadjra firefox nightly. chrome is ok.
